src/repository/DeviceAttributeFileRepository.js

```diff
@@ -27,21 +27,33 @@ class DeviceAttributeFileRepository {
       ...model,
       timestamp: new Date(),
     };
-console.log(model.deviceID);
-    this._fileManager.writeFile(model.deviceID + '.json', modelToSave);
+
+    this._fileManager.writeFile(`${model.deviceID}.json`, modelToSave);
     return modelToSave;
   }
 
   delete(id: string): void {
-    this._fileManager.deleteFile(id + '.json');
+    this._fileManager.deleteFile(`${id}.json`);
   }
 
-  getAll(): Array<DeviceAttributes> {
-    return this._fileManager.getAllData();
+  getAll(userID?: string): Array<DeviceAttributes> {
+    const allData = this._fileManager.getAllData();
+
+    if (userID) {
+      return allData.filter(
+        (attributes: DeviceAttributes): boolean =>
+          attributes.ownerID === userID,
+      );
+    }
+    return allData;
   }
 
-  getById(id: string): DeviceAttributes {
-    return this._fileManager.getFile(id + '.json');
+  getById(id: string, userID?: string): ?DeviceAttributes {
+    const attributes = this._fileManager.getFile(`${id}.json`);
+    if (userID) {
```

Comment:
made `userID` optional, because I need this without userId in one place, and seems `deviceServer` uses these functions, too